wordpress-mobile/WordPress-Android
# Stats - File downloads stats card

Related FluxC changes - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1352

This PR adds a new card - File downloads - to the granular stats screens. The card shows a list of files and the number of downloads in a DESC order by downloads. We only started recording the data after July 29, 2019 so we show a different empty message when the user selects a date before this date. Everything else should work like the other cards.

The file downloads is not supported on Jetpack sites so we hide it there.

I've removed the `RemoteTests` file because it was completely broken.

<img width="490" alt="Screenshot 2019-08-19 at 10 59 22" src="https://user-images.githubusercontent.com/1079756/63254916-e4ac4200-c274-11e9-85db-bca27f7254df.png">
<img width="488" alt="Screenshot 2019-08-19 at 10 58 46" src="https://user-images.githubusercontent.com/1079756/63254918-e4ac4200-c274-11e9-9cd4-54088c273c4a.png">
<img width="486" alt="Screenshot 2019-08-19 at 10 58 25" src="https://user-images.githubusercontent.com/1079756/63254919-e544d880-c274-11e9-8ee7-47e4df341a61.png">

To test:
* Go to WP.com site
* Go to DWMY Stats
* Scroll down
* You see the file downloads card
* Select a date before July 29, 2019
* You see the "unsupported" error message
* Go to a Jetpack site
* Go to DWMY stats
* The File downloads card is not visible

Update release notes:
